### PR TITLE
[irods/irods#2919] Replace ireg -C with -r (main)

### PR DESCRIPTION
**Pull request by alanking**
CI tests are running

Companion to https://github.com/irods/irods/pull/6203

### CODE REVIEW

File: src/ireg.cpp [hunk 1]
```diff
@@ -28,7 +22,7 @@ main( int argc, char **argv ) {
     int nArgv;
 
 
-    optStr = "D:fhkKCG:R:vVZ";
+    optStr = "D:fhkKCrR:vVZ";
```
Comment (alanking) [inline src/ireg.cpp:25]: G is not processed in `parseCmdLineOpt` as it has to do with resource groups which were removed in 4.0.0. I figured it's about time to remove it. I can put it back if anybody disagrees

Comment (trel) [reply src/ireg.cpp:25]: torch approved.

Comment (alanking) [reply src/ireg.cpp:25]: Sweet

**Comment by alanking** [comment]
Will merge along with irods/irods#6203

**Pull request merged**